account_invoice_auto_send_mail/readme/USAGE.rst

```diff
@@ -0,0 +1,4 @@
+It will be necessary that at the level of each newspaper (related to type = sales) we define the fields:
```

Comment:
Frankly, I do not quite understand the explaination here. (i.e., what is "newspaper").
Can you try review it again? Thank you.

Reply:
All the addons had them developed in Spanish and on some occasions I have done the translations automatically to go faster, but in this case it is a big mistake, sorry.